File: js/whosTheOldest.js
// Problem# 197
// Given an object containing the names and ages of a group of people, return the name of the oldest person.
// https://edabit.com/challenge/vPmDvWoP2wj5KpWZi

const oldest = (people) => {
  const oldestAge = Math.max(...Object.values(people));

  return Object.keys(people).find((key) => people[key] === oldestAge);
};

module.exports = oldest;

File: python/whos_the_oldest.py
"""
Who's The Oldest?

Given a dictionary containing the names and ages of a group of people, return the name of the oldest person.

https://edabit.com/challenge/3A6x5GjWmT4t8pssL
"""
from typing import Dict


def oldest(people: Dict[str, int]) -> str:
    return max(people, key=people.get)
